can/interfaces/cantact.py

```diff
@@ -42,28 +44,32 @@ def _detect_available_configs():
             channels.append({"interface": "cantact", "channel": f"ch:{i}"})
         return channels
 
+    @deprecated_args_alias(bit_timing="timing")
     def __init__(
         self,
-        channel,
-        bitrate=500000,
-        poll_interval=0.01,
-        monitor=False,
-        bit_timing=None,
-        _testing=False,
-        **kwargs,
-    ):
+        channel: int,
+        bitrate: int = 500000,
+        poll_interval: float = 0.01,
+        monitor: bool = False,
+        timing: Optional[Union[BitTiming, BitTimingFd]] = None,
```

Comment:
I already commited your suggestion for canalystii, but yes, there actually is a purpose. The `__init__` method might get a `BitTimingFd` object from `can.Bus.__init__` or from configuration. 

So, from a **mypy** point of view, the `__init__` method can receive a `BitTimingFd` instance and must check the type with `isinstance`. With `Optional[BitTiming]`, it would be sufficient to just check for `None`.